src/components/Details.js

```diff
@@ -3,24 +3,24 @@ import React from 'react';
 export default class Details extends React.Component {
     constructor(props) {
         super(props);
-        this.state = {message: 'Hello, this will be the details page for each Movie & TV show'};
+        this.state = { message: 'Hello, this will be the details page for each Movie & TV show' };
     }
 
     componentDidMount() {
         setTimeout(
-            () => this.newMessage(),
+            () => this.comingSoonMessage(),
```

Comment:
Subtle thing! Could you rename it to `setComingSoonMessage`? That would help to describe the action